osu.Game/Screens/Backgrounds/BackgroundScreenDefault.cs

```diff
@@ -22,11 +22,12 @@ public class BackgroundScreenDefault : BackgroundScreen
         private int currentDisplay;
         private const int background_count = 7;
 
-        private string backgroundName => $@"Menu/menu-background-{currentDisplay % background_count + 1}";
+        private string backgroundName;
```

Comment:
Does this need to be a class field? As far as I can tell it could as well be a local inside of `createBackground`.

Reply:
Good point, resolved it in an incoming commit